chart/templates/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+{{- if and (ne .Release.Namespace "kubeflex-system") (not (lookup "v1" "Namespace" "" "kubeflex-system")) }}

```

Comment:
After writing this question, I feel silly even asking it: https://kubernetes.slack.com/archives/C0NH30761/p1765780162847849